cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/sidecar/ArtifactLocalizer.java

```diff
@@ -91,6 +91,10 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ArtifactLocalizer.class);
   private static final String LAST_MODIFIED_HEADER = HttpHeaderNames.LAST_MODIFIED.toLowerCase();
+  // TODO (CDAP-18047) Enable verify SSL cert
+  private static final HttpRequestConfig HTTP_REQUEST_CONFIG = new HttpRequestConfig(15000,
```

Comment:
Please move it to HttpRequestConfig or RemoteClientFactory and indicate it has no SSL in the name